uefi/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # uefi - [Unreleased]
 
+## Added
+- `uefi::system` is a new module that provides freestanding functions for
```

Comment:
nit: In a follow-up: We might should add a `[new api]` "tag" for that in the changelog - this might ease consumers to understand what's going on

Reply:
Agreed, I was thinking maybe just add a short paragraph between `# uefi` and `## Added` .